examples/web/src/components/BusConsole.js

```diff
@@ -0,0 +1,73 @@
+export default {
+    name: 'BusConsole',
+    components: {},
+    data() {
+        return {
+            text: "",
+            text_tmp: "",
+            myToggle: false,
+        }
+    },
+    methods: {
+        bus_timeout: async function(event) {
+            console.log(this.$datas.file);
+            var res = await this.$datas.gstc.bus_timeout("p0", this.$datas.timeout);
```

Comment:
Don't hardcode the pipeline name. Please factor these out and preferably set them as a component parameter.

Reply:
I set a default address and port here. The input label updates with the correct information.